CumulusCI Release 3.49.0 (#3009)
SFDO-Tooling/CumulusCI: HISTORY.rst

History
=======

3.48.3 (2021-12-09)
-------------------

# Critical Changes

> - **Python versions 3.6 and 3.7 are no longer supported.** Please ensure you have an up-to-date version of Python! (#2959)

Reviewer: ```suggestion
- **Python versions 3.6 and 3.7 are no longer supported.** Please ensure you have Python version 3.8, 3.9, or 3.10. (#2959)
```

- The ``dx_convert_from`` task now uses a custom Python task class instead of ``cumulusci.tasks.sfdx.SFDXBaseTask``. In most cases this will have no visible impact, but if you have customized this task config to change the command option, make sure you have also explicitly set its ``class_path`` to ``cumulusci.tasks.sfdx.SFDXBaseTask``. (#2981)
 
# Changes

> - Version information for CumulusCI can now be obtained through a ``--version`` option via ``cci --version``. This output is the same as the ``cci version`` command. (#2974)

Reviewer: ```suggestion
- You can now get the CumulusCI version with ``cci --version``. The output is the same as the ``cci version`` command. (#2974)
```

> - A new feature has been added to the ``robot_libdoc`` task. If you specify the output file as a .csv file, the ``robot_libdoc`` task will generate a csv file with all of the keywords. (#2985)

Reviewer: ```suggestion
- A new feature has been added to the ``robot_libdoc`` task. You can now use ``--f csv`` to generate a CSV file with all of the keywords. (#2985)
```
